substratevm/src/com.oracle.svm.reflect/src/com/oracle/svm/reflect/serialize/SerializationSupport.java

```diff
@@ -113,10 +115,14 @@ public int hashCode() {
 
     private final Map<SerializationLookupKey, Object> constructorAccessors;
 
+    @Platforms(Platform.HOSTED_ONLY.class)
```

Comment:
```suggestion
    @Platforms(Platform.HOSTED_ONLY.class) //
```
To make eclipseformat happy

Reply:
Fixed, also `run mx eclipseformat` to check other format issues.